config/schema/field_types.json

```diff
@@ -1,6 +1,6 @@
 {
   "identifier": {
-    "description": "An identifier, can be used for exact (case and whitespace sensitive) lookup and facetting",
+    "description": "An identifier, can be used for exact (case and whitespace sensitive) lookup and aggregating",
```

Comment:
I think we should use aggregations as the term internally as it describes the used technology, which (for the time being) supporting both `facet_` and `aggreagte_` naming of fields. Once we can move a to versioned API we can look to do a full migration and remove `facet_` naming.